cmd/config/config.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/cri-o/cri-o/lib"
```

Comment:
We need to vendor these dependencies, right?

Reply:
I didn't even mean to have these dependencies...